net-libs/libblkmaker/libblkmaker-0.5.3.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A library implementing getblocktemplate"
+DESCRIPTION="libblkmaker provides an implementation of getblocktemplate, the\
+ current Bitcoin mining protocol."
+HOMEPAGE="https://github.com/bitcoin/libblkmaker"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libblkmaker/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="549193640d4cc7022725754833d9d8a7639c41e1a7d2c5f2588be65857cc0970"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
+    PATCHES="${portVersionedName}.patchset"
+fi
+
+PROVIDES="
+    libblkmaker${secondaryArchSuffix} = $portVersion
+    lib:libblkmaker_0.1${secondaryArchSuffix} = $portVersion
+    lib:libblkmaker_jansson_0.1${secondaryArchSuffix} = $portVersion
+    devel:libblkmaker_0.1${secondaryArchSuffix} = $portVersion
+    devel:libblkmaker_jansson_0.1${secondaryArchSuffix} = $portVersion
```

Comment:
You should add a PROVIDES_devel and a REQUIRES_devel and move the following lines to the PROVIDES_devel bloc:
```
	devel:libblkmaker_0.1$secondaryArchSuffix = $portVersion
	devel:libblkmaker_jansson_0.1$secondaryArchSuffix = $portVersion
```

Reply:
The PROVIDES_devel should start with:
```
PROVIDES_devel="
        libbase58${secondaryArchSuffix}_devel = $portVersion
```
and the REQUIRES_devel should start with:
```
REQUIRES_devel="
        libbase58$secondaryArchSuffix == $portVersion base
```

Reply:
Well, it is OK to drop the documentation on secondary arch because we have x86_gcc2 primary arch.
If you choose to drop support for x86_gcc2 primary arch then you should drop the "if then else fi" and pass $manDir/man3 (instead of $maybe_manDir_man3) in "packageEntries devel":
```
	# devel package
	packageEntries devel \
		$developDir \
		$manDir/man3
```